### PR TITLE
[OperationalInsights] Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by fengzhou-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-## 1.0.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0 (2022-11-29)
```
Comment (Yao725) [inline sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md:3]: Don't we have any changelog in 1.0.0?

Comment (ArthurMa1978) [inline sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md:3]: Please follow https://github.com/Azure/azure-sdk-for-net/pull/32035 to update the changelog.md  and readme.md

**Comment by ArthurMa1978** [comment]
One comments here https://apiview.dev/Assemblies/Review/0ab7afb3131d4eacb1bfc1b0230fece8#0f7e5b49e7834449948add6835edc823

File: sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md (hunk 1 above)
Comment (fengzhou-msft) [reply sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md:3]: Added.

Comment (fengzhou-msft) [reply sdk/operationalinsights/Azure.ResourceManager.OperationalInsights/CHANGELOG.md:3]: Updated

**Review by Yao725 — APPROVED** [review]

**Pull request merged**